bundle/manifests/argoproj.io_argocds.yaml

```diff
@@ -1632,6 +1632,12 @@ spec:
                   For some reason the state of the Argo CD server component could
                   not be obtained.'
                 type: string
+              ssoConfig:
+                description: 'SSOConfig defines the status of SSO configuration. Success:
```

Comment:
we are actually not verifying if the SSO is success or failure ... we are just validating that the configuration provided in the CR is correct.

So lets change the options from `Sucess`, `Failure` and `unknown` to `Valid`, `Invalid` and `unknown` ?

Reply:
The status field is "sso config" not "sso" though.